packages/ovh-manager/ovh-manager.js

```diff
@@ -92,4 +99,46 @@ angular
     _.set(OvhHttpProvider, 'returnSuccessKey', 'data'); // By default, request return response.data
     _.set(OvhHttpProvider, 'returnErrorKey', 'data'); // By default, request return error.data
   })
-  .config(routing);
+  .config(routing)
+  .service('ManagerNavbarService', navbarService)
+  .service('NavbarNotificationService', notificationService)
+  .run(($transitions,
+    $translate,
+    asyncLoader,
+    ouiNavbarConfiguration) => {
+    // $translatePartialLoader.addPart('components');
+
+    const removeOnSuccessHook = $transitions.onSuccess({}, () => {
+      _.set(ouiNavbarConfiguration, 'translations', {
+        notification: {
+          errorInNotification: $translate.instant('common_navbar_notification_error_in_notification'),
+          errorInNotificationDescription: $translate.instant('common_navbar_notification_error_in_notification_description'),
+          markRead: $translate.instant('common_navbar_notification_mark_as_read'),
+          markUnread: $translate.instant('common_navbar_notification_mark_as_unread'),
+          noNotification: $translate.instant('common_navbar_notification_none'),
+          noNotificationDescription: $translate.instant('common_navbar_notification_none_description'),
+        },
+      });
+    });
+
+    // removeOnSuccessHook();
+    import(`./translations/Messages_${$translate.use()}.xml`)
+      .then((module) => {
+        asyncLoader.addTranslations(module.default)
+          .then(() => $translate.refresh())
+          .then(() => removeOnSuccessHook());
+      });
+  })
+  .run(($rootScope, ManagerNavbarService) => {
```

Comment:
$rootScope ... :/

Are you really it's what we want ? :(

Maybe we can use event and controller to avoid using $rootScope, no ?